resources/server/sv_twitter.ts

```diff
@@ -269,6 +287,23 @@ async function doesReportExist(tweetId: number, profileId: number): Promise<bool
   return reports.length > 0;
 }
 
+/**
+ * Check to see if this player has already retweeted this tweet or
+ * is the original poster of a retweeted tweet
+ * @param tweetId - ID of the tweet in question
+ * @param identifier - player identifier of the person performing the retweet
+ */
+async function doesRetweetExist(tweetId: number, identifier: string): Promise<boolean> {
+  const query = `
+  SELECT COUNT(id) as count
+  FROM es_extended.npwd_twitter_tweets
```

Comment:
We probably shouldn't have an explicit DB name here.

Reply:
updated!